ui/page/dcrdex/dcrdex_page.go

```diff
@@ -75,21 +75,31 @@ func (pg *DEXPage) ID() string {
 // Part of the load.Page interface.
 func (pg *DEXPage) OnNavigatedTo() {
 	if !pg.AssetsManager.DEXCInitialized() {
+		pg.showSplashPage = true
 		return
 	}
 
 	if pg.CurrentPage() != nil {
 		pg.CurrentPage().OnNavigatedTo()
+	} else {
+		pg.setPage()
+	}
+}
+
+// setPage starts a goroutine that waits for dexc to get ready before displaying
+// an appropriate page.
+func (pg *DEXPage) setPage() {
```

Comment:
If this is a goroutine that eventually displays the current page, rename to `prepareInitialPage`.